### PR TITLE
Add syllabus to parent

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-12993
affects: Parent
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  One or more files are below the minimum test coverage 50%
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Parent

[MBL-12993](https://instructure.atlassian.net/browse/MBL-12993)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.35% | 93.43% | -0.08%
Core/Core/Syllabus/SyllabusPresenter.swift | 0% | -- | --
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Student/Student/Courses/Syllabus/StudentSyllabusViewController.swift [hunk 1]
```diff
@@ -134,26 +134,24 @@ extension SyllabusViewController: UIScrollViewDelegate {
...
         navigationController?.navigationBar.useContextColor(backgroundColor)
         color = backgroundColor?.ensureContrast(against: .named(.white))
         menu.reload()
     }
 
-    func loadHtml(_ html: String?) {
-        guard let html = html else { return }
+    func updateMenuHeight() {
         menuHeightConstraint.constant = menuHeight
-        syllabus?.loadHTMLString(html, baseURL: nil)
```
Comment (armstrongnate) [inline Student/Student/Courses/Syllabus/StudentSyllabusViewController.swift]: Where does the html get loaded now? Does it?

Comment (bluwave) [reply Student/Student/Courses/Syllabus/StudentSyllabusViewController.swift]: There is now a `Core/SyllabusViewController`  that does the html.  `Core/SyllabusViewController is now a child of this ViewController.  I could change the name of this.  Any ideas on a better name for this viewController?

Comment (armstrongnate) [reply Student/Student/Courses/Syllabus/StudentSyllabusViewController.swift]: Oh gotcha. This VC embeds the Core.SyllabusViewController. I missed that. I think it makes sense. The names are fine with me.

**Pull request merged**